CONTRIBUTING.md

```diff
@@ -45,6 +45,21 @@ tagged [`good first issue`](https://github.com/nabaztag2018/pynab/issues?q=is%3A
 They have been especially selected as they seem not too difficult at first
 sight.
 
+## How-to test a contributor Pull Request (PR)
+
+1. Connect to your Nabaztag 🐰: `ssh pi@nabaztag.local`
+2. Go to the pynab folder: `cd /home/pi/pynab`
+3. List references in a remote repository: `git ls-remote --refs origin`
+4. Open [Pull Request (PR) page](https://github.com/nabaztag2018/pynab/pulls) and memorize the PR number you want to test
+5. Switch to the PR code, here PR #123, change number for your test: `git fetch origin pull/123/head:pr/123 && git checkout pr/123`
+6. Restart Nabaztag deamon: `sudo systemctl restart nabd`
+7. Do your test 🚦
+8. Rollback to default branch: `git checkout release`
+9. Restart Nabaztag deamon: `sudo systemctl restart nabd`
```

Comment:
Step 6. may have to be more complex, depending on the PR contents:
- depending on the code changes, other `nabxxd` daemons may have to be restarted;
- if resources in the Django framework are affected (localization files, DB schema changes involving migration steps,...), it may be needed to run the pynab `upgrade` script.

Same thing in reverse for step 9.
So (even if it may be overkill for 'simple' PRs) the recommendation should be to run the `upgrade`script in all cases.

Reply:
Your suggestion is good, I made a changes in this new PR #177